### ISSUE TITLE
Możliwość pobrania listy produktów z katalogu produktów Allegro

### ISSUE THREAD

**Issue by GregTora** [allegro/allegro-api#9120]
Dzień dobry. Czy jest możliwość pobrania listy produktów z katalogu produktów Allegro? Oczywiście wiem, ze jest ich mnóstwo, jednak zleży mi na zawężeniu wyników do wybranych podkategorii lub po wybranych słowach kluczowych, tj. tak jak działa to w wyszukiwarce? Zależy mi aby była lista zawierała parametry produktu.

**Comment by Lukasz-Zurek** [comment]
Tak, skorzystaj w tym celu z [GET /sale/products](https://developer.allegro.pl/documentation/#operation/getSaleProducts), pamiętając o tym, aby skorzystać z dostępnych parametrów.

Zapoznaj się proszę z [tym poradnikiem](https://developer.allegro.pl/tutorials/jak-jednym-requestem-wystawic-oferte-powiazana-z-produktem-D7Kj9gw4xFA#jak-znalezc-produkt), gdzie cały ten proces jest opisany szczegółowo.
